backend/FwLite/FwLiteShared/TypeGen/ReinforcedFwLiteTypingConfig.cs

```diff
@@ -180,6 +184,14 @@ private static MethodExportBuilder AlwaysReturnPromise(this MethodExportBuilder
         return exportBuilder;
     }
 
+    private static void IgnoreComputedGetters(this PropertyExportBuilder exportBuilder)
+    {
+        // see: https://stackoverflow.com/a/72266104
+        var property = exportBuilder.Member;
+        if (property.SetMethod == null && property.GetMethod?.GetCustomAttribute(typeof(CompilerGeneratedAttribute)) is null)
+            exportBuilder.Ignore();
```

Comment:
Hum, I'm not actually sure I agree with this. When a type with a computed getter is serialized the value will make it into the JSON, if we don't want that then I think it should be JsonIgnored. In the specific example (a class used as input) it maybe doesn't make sense. I'm trying to think about what this type should look like in that case. Maybe the property should just be optional and readonly?